content/en/docs/test.md

```diff
@@ -57,79 +57,99 @@ culpa qui officia deserunt mollit anim id est laborum.
+  otherwise, no matter how many blank lines you put between it and the second.
+
 ### Bullet lists
 
 - This is a list item
 * This is another list item in the same list
 - You can mix `-` and `*`
-  - To make a sub-item, indent two spaces.
-    - This is a sub-sub-item.
-
+    - To make a sub-item, indent two tabstops (4 spaces). **This is different
+      from Jekyll / Kramdown.**
+        - This is a sub-sub-item. Indent two more tabstops (4 more spaces).
+    - Another sub-item.
 
+<!-- separate lists -->
 
-- This is a new list. It has two blank lines from the previous list, If it only
-  had one blank line, it would still show up as part of the previous list.
+- This is a new list. With Hugo, you need to use a HTML comment to separate two
+  consecutive lists. **The HTML comment needs to be at the left margin.**
 - Bullet lists can have paragraphs or block elements within them.
 
-  Just indent the content to be even with the text of the bullet point, rather
-  than the bullet itself.
+      Just indent the content to be one tab stop beyond the text of the bullet
+      point. **This paragraph and the code block line up with the second `l` in
+      `Bullet` above.**
 
-  ```bash
-  ls -l
-  ```
+      ```bash
+      ls -l
+      ```
   
-  - And a sub-list after some block-level content
+      - And a sub-list after some block-level content
+    
 - A bullet list item can contain a numbered list.
-  1.  Numbered sub-list item 1
-  2.  Numbered sub-list item 2
+    1.  Numbered sub-list item 1
+    2.  Numbered sub-list item 2
 
 ### Numbered lists
 
 1.  This is a list item
 2.  This is another list item in the same list. The number you use in Markdown
     does not necessarily correlate to the number in the final output. By
     convention, we keep them in sync.
-3.  For single-digit numbered lists, using two spaces after the period makes
+3.  {{<note>}}
+    For single-digit numbered lists, using two spaces after the period makes
     interior block-level content line up better along tab-stops.
+    {{</note>}}
 
+<!-- separate lists -->
 
-
-1.  This is a new list. It has two blank lines from the previous list, If it only
-    had one blank line, it would still show up as part of the previous list.
+1.  This is a new list. With Hugo, you need to use a HTML comment to separate
+    two consecutive lists. **The HTML comment needs to be at the left margin.**
 2.  Numbered lists can have paragraphs or block elements within them.
 
-    Just indent the content to be even with the text of the bullet point, rather
-    than the bullet itself.
+      Just indent the content to be one tab stop beyond the text of the bullet
+      point. **This paragraph and the code block line up with the `m` in
+      `Numbered` above.**
 
-    ```bash
-    $ ls -l
-    ```
-  
-  - And a sub-list after some block-level content
+      ```bash
+      $ ls -l
+      ```
+      
+    - And a sub-list after some block-level content. This is at the same
```

Comment:
I've tried several different ways of indenting this one and I can't get it to be a bullet inside the numbered list. It always shows up as numbered. It seems like bullet sublists instead of numbered lists should be a thing. I would expect it to work like this:

```markdown
1.  List item 1
    - Sub-item 1
    - Sub-item 2
2.  List item 2
    - Sub-item 1
    - Sub-item 2
3.  List item 3
    - Sub-item 1
    - Sub-item 2
```

1.  List item 1
    - Sub-item 1
    - Sub-item 2
2.  List item 2
    - Sub-item 1
    - Sub-item 2
3.  List item 3
    - Sub-item 1
    - Sub-item 2



cc/ @spf13 @bep

Reply:
This one actually is a CSS problem. Despite the sub-list-item correctly being a `<ul>` instead of the `<ol>`, it is getting a decimal list-style-type for some reason. Trying to work around it in CSS now.